bids/layout/config/derivatives.json

```diff
@@ -49,5 +49,27 @@
             "name": "subset",
             "pattern": "subset-([a-zA-Z0-9]+)"
         }
+    ],
+
+    "default_path_patterns": [
+        "sub-{subject}[/ses-{session}]/{datatype<anat>|anat}/sub-{subject}[_ses-{session}][_acq-{acquisition}][_ce-{ceagent}][_rec-{reconstruction}][_space-{space}][_desc-{desc}]_{suffix<T1w|T2w|T1rho|T1map|T2map|T2star|FLAIR|FLASH|PDmap|PD|PDT2|inplaneT[12]|angio>}{extension<.nii|.nii.gz|.json>|.nii.gz}",
+        "sub-{subject}[/ses-{session}]/{datatype<anat>|anat}/sub-{subject}[_ses-{session}][_acq-{acquisition}][_ce-{ceagent}][_rec-{reconstruction}][_mod-{modality}][_space-{space}][_desc-{desc}]_{suffix<defacemask>}{extension<.nii|.nii.gz|.json>|.nii.gz}",
+        "sub-{subject}[/ses-{session}]/{datatype{anat}|anat}/sub-{subject}[_ses-{session}][_acq-{acquisition}][_ce-{ceagent}][_rec-{reconstruction}][_part-{part}][_space-{space}][_res-{res}][_den-{den}][_label-{label}][_desc-{desc}]_{suffix<mask>}{extension<.nii|.nii.gz|.json>|.nii.gz}",
```

Comment:
I have kept the "part" entity in the path patterns for the mask but it is not there in the other ones: should I
- a) remove it from the mask pattern?
- b) add it to all the others?

I have a preference for "b" because it would stick closer to the spec even if this is an edge case.

Reply:
`part` does not make sense for a mask. A mask is binary, not complex.

Reply:
gotcha

but could it be that some other derivatives may need it?

Reply:
Minimally preprocessed files, I guess, so the ones that are just copies of raw + space/desc. Otherwise probably not. (Fieldmaps already account for it with suffix.) If someone comes up with a use case, they can open an issue. Or hopefully we move away from this onto something schema defined before they have a chance.